docs/evaluation/cache.rst

```diff
@@ -58,33 +56,6 @@ oldest element in the cache is removed when the number of entries becomes
 larger than :py:attr:`~sherpa.models.model.ArithmeticModel.cache` elements (the
 default value for this attribute is 5).
 
-.. _startup-modelcacher1d:
```

Comment:
Is there anywhere else in our documentation that we discuss `startup` and `teardown` (other than them being listed as class methods). If not then we probably should, but where. And is it relevant for this PR.

Reply:
I suggest to address that in the context of #2244. We don't do a lot with those methods right now, though there are potentially some uses in the future - see #2244. Since we don't use them, it's hard to document what their intended use is (and if that use intended when they were added is actually what we would want to use them for today).